packages/super-components/components/msc-network-tile/src/msc-network-tile/components/msc-network-vrack.tsx

```diff
@@ -0,0 +1,171 @@
+        apiClient.v6.get(`/dedicated/server/${this.serviceName}/vrack`),
+        apiClient.v6.get(`/dedicated/server/${this.serviceName}/ips`),
+      ]);
+      this.vrackInfos = vrackResponse.data;
+      this.ipsList = ipsListResponse.data;
+
+      if (this.vrackInfos) {
+        const ipLoadbalancingResponse = await apiClient.v6.get(
+          `/vrack/${this.vrackInfos}/ipLoadbalancing`,
+        );
+        this.ipLoadbalancingList = ipLoadbalancingResponse.data;
+      }
+      if (this.ipsList) {
+        this.ipsList.forEach((ipName) => {
+          apiClient.v6
+            .get(`/ip/${ipName}`)
+            .then((response) => {
+              const ipFOValue = response.data.ip;
+              if (
+                response.data.isAdditionalIp === true ||
+                response.data.type === 'failover'
+              ) {
+                this.ipFOState = ipFOValue;
+              }
+            })
+            .catch((error) => {
+              // Handle errors here
+              console.error(error);
+            });
+        });
+        if (this.ipLoadbalancingList) {
+          this.ipLoadbalancingList.forEach((ipLoadBalancingName) => {
+            apiClient.v6
+              .get(`/ipLoadBalancing/${ipLoadBalancingName}`)
+              .then((response) => {
+                this.natIpInfos = response.data;
+                const natIp = this.natIpInfos?.ip[0];
+                return natIp;
+              });
+          });
+        }
+      } else {
+        // Handle the case where this.ipsList is null or undefined
+        console.error('this.ipsList is null or undefined');
+      }
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  }
+
+  public render() {
+    return (
+      <Host>
+        <div class="container">
+          <span>
+            <div>
+              <osds-text
+                level={ODS_TEXT_LEVEL.heading}
+                size={ODS_TEXT_SIZE._200}
+                color={ODS_THEME_COLOR_INTENT.text}
+              >
+                {this.localeStrings?.server_vrack}
+              </osds-text>
+            </div>
+            <div>
+              <osds-text
+                level={ODS_TEXT_LEVEL.body}
+                size={ODS_TEXT_SIZE._200}
+                color={ODS_THEME_COLOR_INTENT.default}
+              >
+                {this.vrackInfos && this.vrackInfos.length > 0
+                  ? this.vrackInfos.map((info) => <div key={info}>{info}</div>)
+                  : this.localeStrings?.server_vrack_none}
+              </osds-text>
+            </div>
+          </span>
+        </div>
+        <osds-divider separator />
+        {this.ipFOState && (
+          <msc-network-clipboard
+            key="IPFO"
+            clipBoardText="IPFO"
+            clipBoardValue={this.ipFOState}
+            localeStrings={this.localeStrings}
+          />
+        )}
+        {this.ipLoadbalancingList && this.ipLoadbalancingList.length > 0 ? (
```

Comment:
```suggestion
        {this.ipLoadbalancingList?.length > 0 && (
```

a ternary is not needed here

Reply:
Ok but you can remove the ternary though